docs/explore_api/tasks.md

```diff
@@ -0,0 +1,210 @@
+# Tasks
+
+Central to the API's functionality is the concept of a `Task`, which is an 
+abstraction that encapsulates all the information about the
+computational workload you defined as the user. Once you submit a simulation to the 
+API, a `Task` is generated and remains there until you explicitly delete it. This 
```

Comment:
Remove from `and remains there until...` until the final dot. This is not possible at the moment, and I envision that it is not our timeline soon. Try to join with the next phrase.